packages/docusaurus-theme-common/src/utils/jsUtils.ts

```diff
@@ -21,3 +21,13 @@ export function duplicates<T>(
     (v, vIndex) => arr.findIndex((u) => comparator(u, v)) !== vIndex,
   );
 }
+
+/**
+ * Remove duplicate array items (similar to _.uniq)
+ * @param arr The array.
+ * @returns An array with duplicate elements removed by reference comparison.
+ */
+export function uniq<T>(arr: T[]): T[] {
```

Comment:
This is an unrelated change that snuck in, which is fine: moved `uniq` from `utils-common` to `theme-common` because it's only used in `theme-classic`.